### ISSUE TITLE
Project details page

### ISSUE THREAD

**Issue by rimutaka** [stackmuncher/stm_server#8]
Currently projects on dev pages are linked directly to their github pages.

We need to get a quick overview of what those projects are about and zero in on those of interest.

Display that info somewhere somehow once stackmuncher/stm_app#13 is done.

**Comment by rimutaka** [comment]
Need to get feedback

![image](https://user-images.githubusercontent.com/5926028/129461346-969d821e-5728-4f70-bef6-88e704a98ec9.png)

The projects are sorted by date - latest first, projects with < 100 LoC are ignored, only 3 top techs are shown.

**Comment by rimutaka** [comment]
Superseded by #15
